src/sss_client/libwbclient/wbc_sid_sssd.c

```diff
@@ -149,7 +149,9 @@ wbcErr wbcLookupSid(const struct wbcDomainSid *sid,
     if (pdomain != NULL) {
         *pdomain = wbcStrDup(p + 1);
         if (*pdomain == NULL) {
-            wbcFreeMemory(*pname);
+            if (pname != NULL) {
+                wbcFreeMemory(*pname);
+            }
```

Comment:
nitpicking: add empty line after }

Reply:
I didn't find anything relevant in the coding style but ok, I don't mind.

Reply:
Done.